src/controllers/controller.cpp

```diff
@@ -6,21 +6,20 @@
 */
 
 #include <QApplication>
-#include <QScriptValue>
+#include <QJSValue>
 
 #include "controllers/controller.h"
 #include "controllers/controllerdebug.h"
 #include "controllers/defs_controllers.h"
 #include "util/screensaver.h"
 
-Controller::Controller(UserSettingsPointer pConfig)
+Controller::Controller()
         : QObject(),
```

Comment:
Redundant, default ctor is invoked implicitly. Unrelated, but let's remove this stuff.